spec/factories/factories.rb

```diff
@@ -314,7 +314,6 @@
     reviewee { AssignmentTeam.first || association(:assignment_team) }
     reviewer_id 1
     type 'ReviewResponseMap'
-    calibrate_to 0
```

Comment:
I tried to keep it as the original version. I think you can leave it unchanged if it has conflicts with the original one.